app/views/courses/_basic_details_tab.html.erb

```diff
@@ -278,7 +278,7 @@
           <li>GCSE requirements</li>
         </ul>
 
-        <p class="govuk-body">To request other changes to your basic details contact the Becoming a Teacher team:<br /><%= bat_contact_mail_to bat_contact_email_address_with_wrap, subject: "Edit #{course.name} (#{@provider_code}/#{course.course_code})" %></p>
+        <p class="govuk-body">To request other changes to your basic details contact the Becoming a Teacher team:<br /><%= bat_contact_mail_to bat_contact_email_address_with_wrap, subject: "Edit #{course.name} (#{@provider.provider_code}/#{course.course_code})", data: { qa: "course__contact_support" } %></p>
```

Comment:
could we break this line up over multiple lines? I think the reason this bug was also not picked up sooner might have been because you would have to horizontally scroll to see the options.

Reply:
Good spot